internal/data-ingest-cli/main.go

```diff
@@ -47,6 +52,8 @@ func main() {
 	case "otlp-json":
 		fmt.Println("Reading otlpjson data and sending it to collector...")
 		cmd, err := otlpjson.New(otlpjson.Config{
+			SendData:     *sendData,
```

Comment:
instead of checking in the command implementations, should we rather check for `sendData` here and add a dedicated command for just receiving data?